libs/model/src/models/types.ts

```diff
@@ -34,3 +39,16 @@ export type CompositeMap<Parent extends State, Child extends State> = {
   child: ModelStatic<Model<Child>>;
   key: CompositeKey<Parent, Child>;
 };
+
+export type DataTypes = {
```

Comment:
What is the point of this?

Reply:
Sequelize doesn't export types for data types, but exports instances of them instead. There are a few ways to work around this -- `sequelize-typescript` is one of them (https://stackoverflow.com/questions/55166230/sequelize-typescript-typeof-model) but we work around it by referencing `typeof DataTypes` instead. However that information isn't actually exported from the package and since ESM imposes stricter module isolation, the typechecker will throw an error when trying to use `typeof DataTypes` directly now.